update_pmd.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
```

Comment:
```suggestion

# Check if at least one tag has been provided
if [ $# -eq 0 ]; then
    echo "Error: At least one tag is required."
    echo "Usage: update_pmd.sh <tag1> <tag2> ... <tagN>"
    exit 1
fi

```

To make usage very clear even for people that don't like to read 🙈

Reply:
Ok, nice improvement. I was thinking that if someone provides zero versions then the script works perfectly by creating zero tags :sweat_smile:

Reply:
Successfully failed ✅